packages/react-admin/src/mui/button/RefreshButton.js

```diff
@@ -3,8 +3,10 @@ import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import compose from 'recompose/compose';
 import Button from 'material-ui/Button';
+import IconButton from 'material-ui/IconButton';
```

Comment:
I do like the work you have done on the responsiveness of actions but it should be done in a separate PR as it makes this one harder to review. The bulk operations handling is going to be complex so I'd rather keep it separated

Reply:
I understand, do you want it based from next of can I branch from this branch? Saves me some work 👍